server_admin/topics/threat/brute-force.adoc

```diff
@@ -1,84 +1,102 @@
 ====
 
+*Temporary Lockout Parameters*
+
+|===
+|Name |Description |Default
+
+|Wait Increment
+|The time added to the time a user is temporarily disabled when the user's login attempts exceed _Max Login Failures_.
+|1 minute.
+
+|Max Wait
+|The maximum time a user is temporarily disabled.
+|15 minutes.
+
+|Failure Reset Time
+|The time when the failure count resets. The timer runs from the last failed login.
+|12 hours.
+
+|===
+
 *Temporary Lockout Algorithm*
 ====
 . On successful login
 .. Reset `count`
 . On failed login
-.. If time between this failure and the last failure is greater than _Failure Reset Time_
+.. If the time between this failure and the last failure is greater than _Failure Reset Time_
 ... Reset `count`
 .. Increment `count`
-.. Calculate `wait` using _Wait Increment_ * (`count` / _Max Login Failures_). The division is an integer division so will always be rounded down to a whole number
-.. If `wait` equals 0 and time between this failure and the last failure is less than _Quick Login Check Milli Seconds_ then set `wait` to _Minimum Quick Login Wait_ instead
+.. Calculate `wait` using _Wait Increment_ * (`count` / _Max Login Failures_). The division is an integer division rounded down to a whole number
+.. If `wait` equals 0 and the time between this failure and the last failure is less than _Quick Login Check Milliseconds_, set `wait` to _Minimum Quick Login Wait_.
 ... Temporarily disable the user for the smaller of `wait` and _Max Wait_ seconds
 
-Login failures when a user is temporarily disabled do not increment `count`.
+'count` does not increment when a temporarily disabled account commits a login failure.
 ====
 
-The downside of {project_name} brute force detection is that the server becomes vulnerable to denial of service attacks.
-An attacker can simply try to guess passwords for any accounts it knows and these account will be disabled.
-Eventually we will expand this functionality to take client IP address into account when deciding whether to block a user.
+The downside of {project_name} brute force detection is that the server becomes vulnerable to denial of service attacks. When implementing a denial of service attack, an attacker can attempt to log in by guessing passwords for any accounts it knows and eventually causing {project_name} to disable the accounts.
 
-A better option might be a tool like http://www.fail2ban.org/wiki/index.php/Main_Page[Fail2Ban].  You can point this service at the {project_name} server's log file.
-{project_name} logs every login failure and client IP address that had the failure.  Fail2Ban can be used to modify
-firewalls after it detects an attack to block connections from specific IP addresses.
+Consider using intrusion prevention software (IPS). {project_name} logs every login failure and client IP address failure. You can point the IPS to the {project_name} server's log file, and the IPS can modify firewalls to block connections from these IP addresses.
```

Comment:
Is this new content you are suggesting because the current text is not longer valid?

Reply:
yes. I do not think we should be referring to a specific product. I changed the text to refer to the type of product and the changed the rest of the text accordingly.